doc/api/http.md

```diff
@@ -111,6 +111,9 @@ http.get({
 <!-- YAML
 added: v0.3.4
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/34013
+    description: Add `maxTotalSockets` option to agent constructor.
```

Comment:
I don't think this is right. The PR-URL should be the PR that added the functionality in the description field, not this PR that updates the docs only.

Reply:
fixed